service/dap/server.go

```diff
@@ -606,10 +613,10 @@ func (s *Session) handleRequest(request dap.Message) {
 	case *dap.PauseRequest: // Required
 		s.onPauseRequest(request)
 		return
-	case *dap.TerminateRequest: // Optional (capability ‘supportsTerminateRequest‘)
```

Comment:
I guess GoLand doesn't like unicode in comments? I'm ok with this, I just think it's a weird thing to complain about for a linter.

Reply:
This was me actually, I think it's just cleaner to have uniform quotes.